src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -33,14 +34,16 @@ final class LocoProvider implements ProviderInterface
     private HttpClientInterface $client;
     private LoaderInterface $loader;
     private LoggerInterface $logger;
+    private TranslatorBagInterface $translatorBag;
     private string $defaultLocale;
     private string $endpoint;
 
-    public function __construct(HttpClientInterface $client, LoaderInterface $loader, LoggerInterface $logger, string $defaultLocale, string $endpoint)
+    public function __construct(HttpClientInterface $client, LoaderInterface $loader, LoggerInterface $logger, string $defaultLocale, string $endpoint, TranslatorBagInterface $translatorBag)
```

Comment:
this is still a BC break. The parameter must be optional, at least at first (if it's meant to be required - I think so - a deprecation notice should be triggered when null is given).